201-dsc-linux-azure-storage-on-ubuntu/install-resource.parameters.json

```diff
@@ -1,38 +1,39 @@
 {
-  "username": {
-    "value": "yourusername"
-  },
-  "password": {
-    "value": "yourpassword"
-  },
-  "location": {
-    "value": "West US"
-  },
-  "newStorageAccountName": {
-    "value": "uniquestorageaccount"
-  },
-  "vmName": {
-    "value": "yourvmname"
-  },
-  "vmSize": {
-    "value": "Standard_D1"
-  },
-  "ubuntuOSVersion": {
-    "value": "14.04.3-LTS"
-  },
-  "mode": {
-    "value": "Install"
-  },
-  "storageAccountName": {
-    "value": "yourstorageaccountname"
-  },
-  "storageAccountKey": {
-    "value": "yourstorageaccountkey"
-  },
-  "containerName": {
-    "value": "yourcontainername"
-  },
-  "resourceZipFileName": {
-    "value": "yourmodule_version.zip"
-  }
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
```

Comment:
I see that only one value differs between the various parameter file depending on parameter value. Let's use the readme to explain the expected value in each case rather than creating 4 parameter files which will be confusing.